llvm/test/tools/llvm-objcopy/ELF/Inputs/srec-elf-sections.yaml

```diff
@@ -0,0 +1,53 @@
+!ELF
```

Comment:
To inline this with other YAML files in the same test file, I think you'll need to prefix this with `---`